walnuts_cpp/examples/test.cpp

```diff
@@ -66,7 +66,7 @@ void test_nuts(const VectorS& theta_init, G& generator, int D, int N,
     std::cout << "dim " << d << ": mean = " << mean << ", stddev = " << stddev
               << "\n";
   }
-  if (D > 10) {
+  if (D > 5) {
```

Comment:
?

Reply:
The test got changed from only printing 10 to only printing the first 5, but this conditional got missed along the way